### PR TITLE
Allowed to set cookie_domain session in sites.yml

### PR DESCRIPTION
**Pull request by mickaelandrieu**

### CODE REVIEW

**Comment by eric-chau** [comment]
@crouillon, can you review it please?

**Comment by mickaelandrieu** [comment]
Hmm after meeting, don't merge it. I will improve this contribution.

File: Config/services/services.yml [hunk 1]
```diff
@@ -262,6 +263,19 @@ services:
             - default
             - default
 
+    # session
+    session.configurator:
+        class:      %bbapp.session.configurator.class%
+        arguments:  [@bbapp]
+        public:     false
+
```
Comment (crouillon) [inline Config/services/services.yml]: Optionnaly define a service session.handler, for exampe for PDO : 
session.handler:
     class: Symfony\Component\HttpFoundation\Session\Storage\Handler\PdoSessionHandler
     arguments: [@pdo, @config]

File: Security/Listeners/ContextListener.php [hunk 2]
```diff
@@ -43,18 +47,67 @@ public function handle(GetResponseEvent $event)
...
+                    $config = array(
+                        'db_table' => 'session',
+                        'db_id_col' => 'session_id',
+                        'db_data_col' => 'session_value',
+                        'db_time_col' => 'session_time',
+                    );
+                    $storage = new NativeSessionStorage($sessionConfig, new PdoSessionHandler($pdo, $config));
+                }else {
+                    $storage = new NativeSessionStorage($sessionConfig);
+                }
```
Comment (crouillon) [inline Security/Listeners/ContextListener.php]: if ($container->has('session.handler') {
     $storage = new NativeSessionStorage($sessionConfig, $container->gt('session.handler');
}else {
      $storage = new NativeSessionStorage($sessionConfig);
}

**Comment by mickaelandrieu** [comment]
thank you for review @crouillon

**Pull request merged**